source/enforce-macos-app-location.js

```diff
@@ -23,7 +23,7 @@ module.exports = () => {
 		],
 		defaultId: 0,
 		cancelId: 1
-	});
+	}).response;
```

Comment:
You obviously have to remove this now. I thought this was clear.

Reply:
GitHub was showing the change for both lines before I clicked the "approve suggestion" button. Going to manually fix this.